src/redux/features/projects/projectsThunks.ts

```diff
@@ -16,3 +16,17 @@ export const requestProjects = (organizationId: number, locale?: string | null)
     }
   };
 };
+
+export const requestProject = (projectId: number) => async (dispatch: Dispatch, _getState: () => RootState) => {
```

Comment:
When you get a chance, take a look at createAsyncThunk? We have a mixed use-case of async thunks vs non in terraware-web. For the funder-portal we used async thunks across the board. Gave us better request status management.

Reply:
If you'd like, up to you. Or leave it for future requests.